schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/cart/ShoppingCart.java

```diff
@@ -185,4 +246,56 @@ public void setData(Object data) {
             this.data = data;
         }
     }
+
+    public static class PaymentOption implements Serializable {
+        private PaymentTypeEnum paymentType;
```

Comment:
The solution would be more elegant if PaymentOption were an enum IMO.
Either with interface
~~~java
public interface Operator {
    int apply (int a, int b);
}

public enum SimpleOperators implements Operator {
    PLUS { 
        int apply(int a, int b) { return a + b; }
    },
    MINUS { 
        int apply(int a, int b) { return a - b; }
    };
}
~~~
Or abstract methods

~~~java
public enum Animal {
    CAT {
        public String makeNoise() { return "MEOW!"; }
    },
    DOG {
        public String makeNoise() { return "WOOF!"; }
    };

    public abstract String makeNoise();
}
~~~

Specifically
~~~java
enum PaymentOption {
PayPal {
  BigDecimal getFee(Double total){
   return ...;
 }

 abstract BigDecimal getFee(Double total);
~~~

Of course the only problem would be it is not enabled/disabled...

So, I guess we keep it like this for now...